script/test

```diff
@@ -358,13 +358,22 @@ do_build_otbr_docker()
         if [[ -z ${LOCAL_OTBR_DIR} ]]; then
             ./script/git-tool clone https://github.com/openthread/ot-br-posix.git --depth 1 "${otbrdir}"
         else
-            cp -r "${LOCAL_OTBR_DIR}"/* "${otbrdir}"
-            rm -rf "${otbrdir}"/build
+            rsync -r \
```

Comment:
That's a useful way of cloning a local repo. However, it will only clone committed changes. A developer may want to include unstaged changes/files in the docker image, otherwise the result may be confusing.